test/parallel/test-assert.js

```diff
@@ -409,6 +409,11 @@ assert.throws(() => {
 }, /Got unwanted exception: user message/,
               'a.doesNotThrow ignores user message');
 
+assert.throws(() => {
+  assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
+}, /Got unwanted exception: user message\ntest/,
+              'a.doesNotThrow ignores message of the unwanted exception');
```

Comment:
Yeah I've seen that happen before, it wants to align with the function call's opening `(`.